static/app/routes.tsx

```diff
@@ -2034,15 +2044,6 @@ function routes() {
         <Route path="/:orgId/" component={errorHandler(OrganizationDetails)}>
           <Route component={errorHandler(OrganizationRoot)}>
             {hook('routes:organization-root')}
-            <Route
-              path="/organizations/:orgId/stats/"
-              componentPromise={() =>
-                import(
-                  /* webpackChunkName: "OrganizationStats" */ 'app/views/organizationStats'
```

Comment:
Does your view rely on `organization.projects` or is it already using `withProjects`?

Reply:
Yep we're using `withProjects`.

Reply:
The `usageStatsProjects` component looks for `loadingProjects`being passed from `withProjects` too, so the loading state is handled.